package.json

```
@@ -6,40 +6,12 @@
   "main": "hardhat.config.ts",
   "private": true,
   "scripts": {
-    "test": "npx hardhat test",
+    "test": "hardhat test",
     "test:coverage": "hardhat coverage",
-    "lint": "npx hardhat check",
+    "lint": "hardhat check",
     "compile": "hardhat compile",
-    "deploy:mainnet": "hardhat --network mainnet deploy",
-    "deploy:ropsten": "hardhat --network ropsten deploy",
-    "deploy:rinkeby": "hardhat --network rinkeby deploy",
-    "deploy:goerli": "hardhat --network goerli deploy",
-    "deploy:fantom": "hardhat --network fantom deploy",
-    "deploy:celo": "hardhat --network celo deploy",
-    "deploy:avalanche_test": "hardhat --network avalanche_test deploy",
-    "deploy:avalanche": "hardhat --network avalanche deploy",
-    "deploy:arbitrum_rinkeby": "hardhat --network arbitrum_rinkeby deploy",
-    "deploy:arbitrum": "hardhat --network arbitrum deploy",
-    "deploy:optimism": "hardhat --network optimism deploy",
-    "deploy:optimism_kovan": "hardhat --network optimism_kovan deploy",
-    "deploy:xdai": "hardhat --network xdai deploy",
-    "deploy:bsc_test": "hardhat --network bsc_test deploy",
-    "deploy:bsc_mainnet": "hardhat --network bsc_mainnet deploy",
-    "deploy:matic_mainnet": "hardhat --network matic_mainnet deploy",
-    "deploy:aurora_test": "hardhat --network aurora_test deploy",
-    "deploy:aurora": "hardhat --network aurora deploy",
-    "deploy:fuse": "hardhat --network fuse deploy",
-    "deploy:boba": "hardhat --network boba deploy",
-    "deploy:boba_test": "hardhat --network boba_test deploy",
-    "deploy:metis": "hardhat --network metis deploy",
-    "deploy:metis_test": "hardhat --network metis_test deploy",
-    "deploy:moonriver": "hardhat --network moonriver deploy",
-    "deploy:cfx": "hardhat --network conflux_eSpace deploy",
-    "deploy:cfx_test": "hardhat --network conflux_eSpace_test deploy",
-    "deploy:harmony": "hardhat --network harmony deploy",
-    "deploy:harmony_test": "hardhat --network harmony_test deploy",
-    "deploy:kardia": "hardhat --network kardia deploy",
-    "prettier:solidity": "./node_modules/.bin/prettier --write contracts/**/*.sol",
+    "deploy": "hardhat deploy --network",
+    "prettier:solidity": "prettier --write contracts/**/*.sol",
     "format": "prettier --write './test/*.js' './deploy/*.ts'"
```

```suggestion
    "pretter": "prettier --write .",
```

1. merge pretter commands
2. use `.prettierignore` ignore files
    see https://prettier.io/docs/en/ignore.html#ignoring-files-prettierignore